sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java

```diff
@@ -120,6 +120,13 @@ private CorePropertyDefinitions() {
         .multiValues(true)
         .defaultValue(CoreProperties.PREVIEW_EXCLUDE_PLUGINS_DEFAULT_VALUE)
         .build(),
+      PropertyDefinition.builder(CoreProperties.SKIP_ONBOARDING_TUTORIAL)
+        .name("Skip the onboarding tutorial")
+        .type(PropertyType.BOOLEAN)
+        .description("By default, each new user is lead through a tutorial, that explains how to analyze a first project, after logging in for the fist time. By enabling this setting, the tutorial will not be shown automatically, but only be visible in the help.")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 264 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)